xdebug_com.c

```diff
@@ -21,169 +21,242 @@
 #include <errno.h>
```

Comment:
You're changing the permissions from rw-r--r-- to rwx-r-xr-x for this file. Please don't do that.